helms/odahu-flow-opa/policies/permissions.rego

```diff
@@ -8,14 +8,22 @@ permissions := {
     	[".*", "api/v1/model/packaging.*"],
     	[".*", "api/v1/model/training.*"],
     	["GET", "api/v1/configuration.*"],
-    	["GET", "api/v1/connection.*"],
+    	["GET", "api/v1/connection/*[a-zA-Z0-9-]*/*$"],
```

Comment:
Should this regex match strings like "api/v1/connection/s0m3-CoNnectioN-8/" (with optional "/" ending)?

Reply:
yes